toonz/sources/toonzlib/preferences.cpp

```diff
@@ -373,7 +373,17 @@ Preferences::Preferences()
   setUndoMemorySize(m_undoMemorySize);
   m_blankColor = TPixel32(r, g, b);
 
+  getValue(*m_settings, "pixelsOnly", m_pixelsOnly);
+  
+  //for Pixels only 
   QString units;
+  units                    = m_settings->value("oldUnits").toString();
+  m_oldUnits = units;
+
+  units                    = m_settings->value("oldCameraUnits").toString();
+  m_oldCameraUnits = units;
```

Comment:
Please add the default value as follows in case the value is not found in the preference file when the first launch of the software:
```
units                    = m_settings->value("oldUnits", m_units).toString();
m_oldUnits = units;

units                    = m_settings->value("oldCameraUnits", m_cameraUnits).toString();
m_oldCameraUnits = units;
```